packages/angular/projects/ui-angular/src/lib/primitives/error/error.component.html

```diff
@@ -21,7 +21,6 @@
   </div>
   <button
     amplify-button
-    class="amplify-field-group__control amplify-alert__dismiss"
```

Comment:
FYI for context: https://github.com/aws-amplify/amplify-ui/pull/3861/files

I created a ticket to handle classname change in another PR.